### PR TITLE
Dobot

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Fix linting
- Refactor code
- Delete old files

### CODE REVIEW

File: notebooks/500_use_cases/501_dobot/501a_Dataset_creation_and_Inference_with_a_robotic_arm.ipynb [hunk 1]
```diff
@@ -47,45 +47,31 @@
   },
```
Comment (paularamo) [inline notebooks/500_use_cases/501_dobot/501a_Dataset_creation_and_Inference_with_a_robotic_arm.ipynb:47]: <pre class="ql-syntax" spellcheck="false">Line #23.            image_name = str(Path(f"{path}" + "/input_" + str(now.strftime("%Y%m%d%H%M%S")) + ".jpg"))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (paularamo) [inline notebooks/500_use_cases/501_dobot/501a_Dataset_creation_and_Inference_with_a_robotic_arm.ipynb:47]: <pre class="ql-syntax" spellcheck="false">Line #29.            image_name = str(Path(f"{path}" + "/input_" + str(now.strftime("%Y%m%d%H%M%S")) + ".jpg"))
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (paularamo) [inline notebooks/500_use_cases/501_dobot/501a_Dataset_creation_and_Inference_with_a_robotic_arm.ipynb:47]: <pre class="ql-syntax" spellcheck="false">Line #100.                    filename = filename_fc(folder, dataset_path)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: notebooks/500_use_cases/501_dobot/501b_Training_a_model_with_cubes_from_a_robotic_arm.ipynb [hunk 2]
```diff
@@ -37,7 +37,7 @@
   },
```
Comment (paularamo) [inline notebooks/500_use_cases/501_dobot/501b_Training_a_model_with_cubes_from_a_robotic_arm.ipynb:37]: <p>If we need to show the output of the cells, I suggest to show just the config file, and the image of one cube and the images with results.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/927/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: requirements/extras.txt [hunk 3]
```diff
@@ -2,4 +2,5 @@ comet-ml>=3.31.7
 gradio>=2.9.4
 tensorboard
 wandb==0.12.17
-ipykernel
\ No newline at end of file
+GitPython
```
Comment (djdameln) [inline requirements/extras.txt:5]: What is this used for?

Comment (ashwinvaidya17) [reply requirements/extras.txt:5]: There is this line `from git.repo import Repo` which requires gitpython

**Pull request merged**